File: app/src/main/java/io/twere/plainpure/data/model/dribbble/Follower.java
package io.twere.plainpure.data.model.dribbble;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.Date;

public class Follower implements Serializable {

  @SerializedName("id") private long id;

  @SerializedName("created_at") private Date createdAt;

  @SerializedName("follower") private User user;

  public User getUser() {
    return user;
  }

  public void setUser(User user) {
    this.user = user;
  }

  public Date getCreatedAt() {
    return createdAt;
  }

  public void setCreatedAt(Date createdAt) {
    this.createdAt = createdAt;
  }

  public long getId() {
    return id;
  }

  public void setId(long id) {
    this.id = id;
  }
}

File: app/src/main/kotlin/io/twere/plainpure/data/local/follower/Follower.kt
package io.twere.plainpure.data.local.follower

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import io.twere.plainpure.data.local.user.User
import java.util.Date

data class Follower(
    @SerializedName("id") @Expose var id: Long,
    @SerializedName("created_at") @Expose var createdAt: Date,
    @SerializedName("follower") @Expose var follower: User) {
}